geometry_msgs/Point GraspingPoint
geometry_msgs/Point ReferencePoint
geometry_msgs/Point[] PlanePoints
